### PR TITLE
Refactor the BannerWithImage component

### PR DESCRIPTION
**Pull request by mark-a-wilson**
This addresses the issues noted in #375:

- Fixes position of banner text with spacing more aligned with Figma designs
- Improves responsiveness and stability of the banner image by passing image into the `background-image` property of the container
- Reduces vertical space taken up on mobile displays
- Slightly tweak position of navigation to unhide the yellow border of the header (more to do here in future refactoring)

### CODE REVIEW

File: react-frontend/src/components/StyleComponents/bannerWithImage.js [hunk 1]
```diff
@@ -83,16 +68,17 @@ export const BannerTitle = styled.h1.attrs({
 export const BannerWithImageWrapper = styled.div.attrs({
   className: 'banner',
 })`
-  height: 600px;
-  margin-bottom: 120px;
-  overflow: hidden;
-  position: relative;
+  background-image: url(${(props) => props.backgroundImage});
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/StyleComponents/bannerWithImage.js:71]: This was the one I was looking at and wondering if ` url(${ props.backgroundImage});` would actually work?

**Review by thegentlemanphysicist — COMMENTED** [review]
Just one possible simplifying line that I'm curious about

File: react-frontend/src/components/StyleComponents/bannerWithImage.js (hunk 1 above)
Comment (mark-a-wilson) [reply react-frontend/src/components/StyleComponents/bannerWithImage.js:71]: I tried — it doesn't seem to work:

image

**Comment by mark-a-wilson** [comment]
Too much dependent stuff going on in this PR. Have replaced with #393.

**Pull request closed without merging**